aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -1112,6 +1125,13 @@ void impl_func_prod(
   }
 }
 
+Tensor prod(const Tensor& self, int64_t dim, bool keepdim, c10::optional<ScalarType> opt_dtype) {
```

Comment:
Is this change needed in this PR?
The function `prod(Tensor self, int dim, bool keepdim=False, *, ScalarType? dtype=None) -> Tensor` should be autogenerated with https://github.com/pytorch/pytorch/blob/54d060a8c91c27167925b380698331850352c841/aten/src/ATen/native/native_functions.yaml#L4200